src/features/canvassAssignments/hooks/useSidebarStats.ts

```diff
@@ -134,6 +135,7 @@ export default function useSidebarStats(
     stats,
     sync: () => {
       dispatch(visitsInvalidated(assignmentId));
+      dispatch(placesInvalidated);
```

Comment:
This is not dispatching the action, but a function.

```suggestion
      dispatch(placesInvalidated());
```